src/util/Expected.h

```diff
@@ -0,0 +1,264 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of clio: https://github.com/XRPLF/clio
+    Copyright (c) 2023, the clio developers.
+    Copyright (c) 2021 Ripple Labs Inc.
```

Comment:
delete this line?

Reply:
That would violate `rippled`s requirement to keep this notice when copying the code. I added this line on purpose :)